websites/A/AnimesOnline/presence.ts

```diff
@@ -68,6 +84,22 @@ presence.on("UpdateData", async () => {
 		presenceData.details =
 			document.querySelector("div.data > h1")?.textContent ?? "desconhecido";
 		presenceData.state = "Selecionando um episódio";
+		presenceData.smallImageKey = "reading";
+		if (cover) {
+			const name = document.querySelector(
+					"#single > div.content.right > div.sheader > div.data > h1"
+				).textContent,
+				ccover = document.querySelector<HTMLImageElement>(
+					"#single > div.content.right > div.sheader > div.poster > img"
+				).src;
+			if (
+				getCookie("PMD_prevACover") !== ccover &&
+				getCookie("PMD_prevAName") !== name
+			) {
+				setCookie("PMD_prevAName", `${name}`, 1);
+				setCookie("PMD_prevACover", `${ccover}`, 1);
+			} else presenceData.largeImageKey = `${getCookie("PMD_prevACover")}`;
```

Comment:
why cookies? cant you just save them outside updatedata?

Reply:
Your change doesn't fix or respond to this...

Reply:
*โปรดอธิบายทำไมต้องใช้ cookie*

Reply:
Because if going into new web link but in the same url, the image will not show anything anime cover pic, so it must be used. Cookies to save image links for showing images like the same anime cover